rpc/src/transaction_status_service.rs

```diff
@@ -37,39 +37,59 @@ impl TransactionStatusService {
         let thread_hdl = Builder::new()
             .name("solTxStatusWrtr".to_string())
             .spawn(move || {
-                info!("TransactionStatusService has started");
```

Comment:
nit: Let's keep the log, these aren't in every thread but a decent number of them have `service starting` and `service stopping` logs like this